twilight-http/src/routing.rs

```diff
@@ -4612,7 +4488,7 @@ mod tests {
 
         assert_eq!(
             route.to_string(),
-            format!("guilds/{GUILD_ID}/members/search?query=foo%20bar&limit=99")
+            format!("guilds/5/members/search?limit=99&query=foo%20bar")
```

Comment:
Don't inline `GUILD_ID`
```suggestion
            format!("guilds/{GUILD_ID}/members/search?limit=99&query=foo%20bar")
```